cmd/ebrelayer/events/event.go

```diff
@@ -17,23 +17,34 @@ import (
 
 // LockEvent : struct which represents a single smart contract event
 type LockEvent struct {
-	Id    [32]byte
-	From  common.Address
-	To    []byte
-	Token common.Address
-	Value *big.Int
-	Nonce *big.Int
+	Chain    string
+	Contract common.Address
+	Id       [32]byte
+	From     common.Address
+	To       []byte
+	Token    common.Address
+	Symbol   string
+	Value    *big.Int
+	Nonce    *big.Int
 }
 
 // NewLockEvent : parses LogLock events using go-ethereum's accounts/abi library
-func NewLockEvent(contractAbi abi.ABI, eventName string, eventData []byte) LockEvent {
+func NewLockEvent(contractAbi abi.ABI, contractAddress string, eventName string, eventData []byte) LockEvent {
 	// Check event name
 	if eventName != "LogLock" {
 		log.Fatal("Only LogLock events are currently supported.")
 	}
 
 	// Parse the event's attributes as Ethereum network variables
 	event := LockEvent{}
+
+	if !common.IsHexAddress(contractAddress) {
+		log.Fatalf("Only Ethereum contracts are currently supported. Invalid address: %v", contractAddress)
+	}
+
+	event.Chain = "Ethereum"
```

Comment:
Some comments:
 - Can we use ChainID, rather than chain, representing the ID based on EIP-155 (https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md)
 - We can likely get the ChainID from the geth client (client.ChainID)

Reply:
https://github.com/swishlabsco/peggy/pull/4/commits/976aa67621a3fddc4994c9abc7e792104ffc8c05